### PR TITLE
block_hotplug_negative: set image_format for hotplug block device

### PR DESCRIPTION
**Pull request by aliang123**
Default image_format for hotplug device is "qcow2",
but for raw image, it should be raw, so error occurs.
Correct it.

Signed-off-by: Aihua Liang <aliang@redhat.com>

bug id:1550397

### CODE REVIEW

**Comment by aliang123** [comment]
Test cmd:
 python ConfigTest.py --guestname=RHEL.7.5 --platform=x86_64 --verbose=yes --driveformat=virtio_blk --testcase=block_hotplug_negative --imageformat=raw
Test result:
PASS 1-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.raw.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.block_hotplug_negative
PASS 2-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.raw.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.block_hotplug_negative.data_plane

Test cmd:
  python ConfigTest.py --guestname=RHEL.7.5 --platform=x86_64 --verbose=yes --driveformat=virtio_blk --testcase=block_hotplug_negative
Test result:
PASS 1-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.block_hotplug_negative
PASS 2-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.block_hotplug_negative.data_plane

**Comment by CongLi** [comment]
It's better to specify 'Default image_format' --> 'Default img_format_type' in commit log. image_format is confused.

**Comment by CongLi** [comment]
Ack.

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by xuwei@redhat.com

**Comment by CongLi** [comment]
Ack.

**Comment by CongLi** [comment]
why not set image_format in    drive.set_param("format", params["image_format"]) ?

**Comment by CongLi** [comment]
the additional disk create and boot should work even the default type is qcow2 although system_disk is raw.

**Comment by CongLi** [comment]
Ack.

File: qemu/tests/block_hotplug_negative.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def drive_unplug_plug(drive, vm):
         drive.hotplug(vm.monitor)
 
     img_list = params.get("images").split()
-    img_format_type = params.get("img_format_type", "qcow2")
+    img_format_type = params.get("img_format_type", params["image_format"])
```
Comment (luckyh) [inline qemu/tests/block_hotplug_negative.py]: Please do not use the additional param if there is no valid reason.

**Comment by CongLi** [comment]
@aliang123 
Could you please also update the commit log based on your latest code change?

**Comment by aliang123** [comment]
Close this patch for conflict, will send a new one.

**Pull request closed without merging**